solvis/__init__.py

```diff
@@ -1,29 +1,15 @@
-"""The base Solvis analysis package.
+"""This package exports the **Solvis** analysis library.
+
```

Comment:
remove the word "seismic" from the description as that implies to a geophysicist that you are analysing seismic waveform data.

Proposed:
```
This library is designed to facilitate analysis and visualization of OpenSHA inversion solutions.
It includes methods for filtering based on rupture rate, geographic location, and fault involvment.
Results can be exported to multiple formats such as GeoJSON.
```